scripts/cd/push-bundle-and-index-image.sh

```diff
@@ -76,13 +76,18 @@ handle_missing_version_in_combined_repo() {
     fi
 }
 
+# get abs path of the directory where this script is located
+# this way the source of the local file can work independently of where the script is executed from.
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 # use the olm-setup as the source
-OLM_SETUP_FILE=scripts/cd/olm-setup.sh
+OLM_SETUP_FILE=${SCRIPT_DIR}/olm-setup.sh
 OWNER_AND_BRANCH_LOCATION=${OWNER_AND_BRANCH_LOCATION:-codeready-toolchain/toolchain-cicd/master}
 
 if [[ -f ${OLM_SETUP_FILE} ]]; then
+    echo "sourcing local olm setup file ${OLM_SETUP_FILE}"
     source ${OLM_SETUP_FILE}
```

Comment:
this won't work if the script is being pulled from the GH repository. 
But this could work:
```bash 
SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
# use the olm-setup as the source
OLM_SETUP_FILE=scripts/cd/olm-setup.sh
LOCAL_OLM_SETUP_FILE=${SCRIPT_DIR}/olm-setup.sh
OWNER_AND_BRANCH_LOCATION=${OWNER_AND_BRANCH_LOCATION:-codeready-toolchain/toolchain-cicd/master}

if [[ -f ${LOCAL_OLM_SETUP_FILE} ]]; then
    echo "sourcing local olm setup file ${LOCAL_OLM_SETUP_FILE}"
    source ${LOCAL_OLM_SETUP_FILE}
```

Reply:
correct! fixing it! thanks!